recipe/recipe.yaml

```diff
@@ -0,0 +1,64 @@
+context:
+  name: fparser
+  version: 0.1.4
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.io/packages/source/${{ name[0] }}/${{ name }}/${{ name }}-${{ version }}.tar.gz
+  sha256: 00d4f7e9bbd8a9024c3c2f308dd3be9b0eeff3cb852772c9f3cf0c4909dbafd4
+
+build:
+  python:
+    entry_points:
+    - fparser2=fparser.scripts.fparser2:main
+  noarch: python
+  number: 1
+  script:
+    content: $PYTHON -m pip install . --no-deps --ignore-installed -vv
+    env:
+      SETUPTOOLS_SCM_PRETEND_VERSION: ${{ version }}
+      
+requirements:
+  host:
+    - pip
+    - setuptools
+    - setuptools-scm
+    - setuptools_scm_git_archive
+    - python >=2.7
+  run:
+    - python >=2.7
+    - setuptools-scm
+
+tests:
+  - python:
+      imports:
+      - fparser
+      - fparser.common
+      - fparser.common.tests
+      - fparser.one
+      - fparser.one.tests
+      - fparser.scripts
+      - fparser.two
+  - script:
+    - fparser2 --help
+    # check the version
+    - python -c "import fparser; print(fparser.__version__); assert fparser.__version__ == '${{ version }}'"
+
+
+about:
+  homepage: https://github.com/stfc/fparser
+  license: BSD-3-Clause
+  license_file: LICENSE
+  summary: Modern Fortran Parser
+  documentation: https://github.com/stfc/fparser
+  repository: https://github.com/stfc/fparser
+
+extra:
+  recipe-maintainers:
+    - arporter
+    - matthewrmshin
+    - MatthewHambley
+    - rupertford
```

Comment:
Please could you remove Rupert Ford from this list. He is sadly no longer with us :-(

Reply:
I have just done this, as requested.